### PR TITLE
text editor

### LINKED ISSUE #43: Add Markup language
Markup language for links, pictures, new lines in blog posts - general formatting on the page. Consider allowing html injection.

### PR DESCRIPTION
**Pull request by Vutov**
resolves #43

### CODE REVIEW

File: src/Steppenwolf/Startup.cs [hunk 1]
```diff
@@ -57,7 +57,8 @@ public void ConfigureServices(IServiceCollection services)
                 .AddRoleManager<RoleManager<IdentityRole>>()
                 .AddEntityFrameworkStores<PostgresDbContext>();
 
-            services.AddServerSideBlazor(); // TODO change to client-side
+            services.AddServerSideBlazor() // TODO change to client-side
```
Comment (vidolch) [inline src/Steppenwolf/Startup.cs:63]: Can't we move this into:
#if DEBUG
#else
#endif

File: src/Steppenwolf/Steppenwolf.csproj [hunk 2]
```diff
@@ -25,4 +25,10 @@
     <ProjectReference Include="..\Steppenwolf.Services\Steppenwolf.Services.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (vidolch) [inline src/Steppenwolf/Steppenwolf.csproj]: Do we need this?

File: src/Steppenwolf/Startup.cs (hunk 1 above)
Comment (Vutov) [reply src/Steppenwolf/Startup.cs:63]: I'll see if it is possible to use environment setting here, don't particularly like the if DEBUG

File: src/Steppenwolf/Steppenwolf.csproj (hunk 2 above)
Comment (Vutov) [reply src/Steppenwolf/Steppenwolf.csproj]: Good catch, will remove.

**Review by vidolch — APPROVED** [review]

**Pull request merged**